quantuminspire/util/api/remote_runtime.py

```diff
@@ -59,23 +62,33 @@ def __init__(self) -> None:
         host = "https://staging.qi2.quantum-inspire.com"
         self._configuration = Configuration(host=host, api_key={"user": str(settings.auths[host]["user_id"])})
 
-    def run(self, program: BaseAlgorithm) -> BatchRun:
+    def run(self, program: BaseAlgorithm) -> int:
         """Execute provided algorithm/circuit."""
         return asyncio.run(self._create_flow(program))
 
-    def get_results(self) -> None:
+    async def _get_results(self, run_id: int) -> Any:
```

Comment:
why a returntype of  Any and not Optional[List[Result]]

Reply:
Same reason as above, return type needs to apply to all runtime types and the structure returned by the remote runtime isn't necessarily the same as a local variant